docs/sources/menu.yaml

```diff
@@ -255,28 +255,33 @@
       name: Overview
     - link: /variables/variable-examples/
       name: Variable examples
-    - link: /variables/add-query-variable/
-      name: Add query variable
-    - link: /variables/add-custom-variable/
-      name: Add custom variable
-    - link: /variables/add-text-box-variable/
-      name: Add text box variable
-    - link: /variables/add-constant-variable/
-      name: Add constant variable
-    - link: /variables/add-data-source-variable/
-      name: Add data source variable
-    - link: /variables/add-interval-variable/
-      name: Add interval variable
-    - link: /variables/add-ad-hoc-filters/
-      name: Add ad hoc filters
+    - name: Variable types
+      link: /variables/variable-types/
+      children:
+      - link: /variables/variable-types/add-query-variable/
+        name: Add query variable
```

Comment:
Do we need Add in the name for every page here? I can get it's ok to have it in the page title for SEO reasons but in the nav it feels a bit superfluous

Reply:
Yes, we do. We don't know which variable a user wants to know about, so we want to make this content as discoverable as possible for the user.

Reply:
@scoren-gl will let me know if I am wrong. :)

Reply:
Also, the verb in the title signals to the reader that this is a TASK topic with instructions on HOW to do the task, in this case adding the variable.